src/UITests/LoadTestFile.cs

```diff
@@ -10,35 +12,45 @@ namespace UITests
     [TestClass]
     public class LoadTestFile : AIWinSession
     {
+        static readonly string TestFileName = "WildlifeManagerTest.a11ytest";
+        static readonly string TestFilePath = $"{Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)}\\TestFiles";
+
         [TestMethod]
         [TestCategory("NoStrongName")]
-        public void VerifyResultsInUIATreePage()
+        public void LoadTestFileTests()
```

Comment:
>LoadTestFileTests [](start = 20, length = 17)

How can we make clearer what this method is? Better name and/or comments?

Reply:
I'm open to suggestions for the name. I went with <name of test class>Tests because that seemed like the most succinct name for the method that executes the test class's tests. 

A comment could be helpful--something like "The method that executes all of this test class's tests"? Or do we want to add an explanation of why we're taking this approach: "In order to better control order of test execution, this method is the only TestMethod in this class and calls methods which run the actual test arrange/act/assert code"?

Reply:
maybe a comment / name with the term(s) 'test scenario entry point'?